apps/web/store/archive-queue.ts

```
@@ -1,103 +1,146 @@
-      });
-
-      refetch?.();
-    }),
+    Object.entries(threads).map(
+      ([_key, { threadId, actionType, labelId }]) =>
+        async () => {
+          await pRetry(
+            async (attemptCount) => {
+              console.log(
+                `Queue: ${actionType}. Processing ${threadId}` +
+                  (attemptCount > 1 ? ` (attempt ${attemptCount})` : ""),
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)